third_party/python.bzl

```diff
@@ -71,3 +71,13 @@ def tensorboard_python_workspace():
       strip_prefix = "six-1.10.0",
       build_file = str(Label("//third_party:six.BUILD")),
   )
+
+  native.new_http_archive(
+      name = "org_pythonhosted_portpicker",
+      urls = [
+          "https://pypi.python.org/packages/96/48/0e1f20fdc0b85cc8722284da3c5b80222ae4036ad73210a97d5362beaa6d/portpicker-1.1.1.tar.gz",
```

Comment:
Please make `http://mirror.bazel.build/pypi.python.org/packages/96/48/0e1f20fdc0b85cc8722284da3c5b80222ae4036ad73210a97d5362beaa6d/portpicker-1.1.1.tar.gz` first in the list.